darkdetect/_linux_detect.py

```diff
@@ -4,25 +4,20 @@
 #  Distributed under the terms of the 3-clause BSD License.
 #-----------------------------------------------------------------------------
 
-from ctypes import util, cdll, c_char_p, byref
+import subprocess
 
 
 def theme():
     # Here we just triage to GTK settings for now
-    try:
-        gtklib = cdll.LoadLibrary(util.find_library('gtk-3'))
-        gtklib.gtk_init(None, None)
-        settings = gtklib.gtk_settings_get_default()
-        res = c_char_p()
-        gtklib.g_object_get(settings, b"gtk-theme-name", byref(res), 0)
-        theme = res.value.decode()
-    except Exception:
-        return 'Light'
+    out = subprocess.Popen(
+        ['gsettings', 'get', 'org.gnome.desktop.interface', 'gtk-theme'],
+        stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout, _ = out.communicate()
+    theme = stdout.lower().decode().strip()[1:-1]  # remove start and end quote
```

Comment:
I would probably wrap this in a `try`/`except` block for all the cases in which `stdout` is an unexpected value or even `None`.

Reply:
At least for `run` I don't think it's ever "None", just an empty bytes array
```
>>> out = subprocess.run('which', capture_output=True)
>>> out
CompletedProcess(args='which', returncode=1, stdout=b'', stderr=b'')
```
But I'll wrap the `decode` part just to be safe